backend/app/engine/projects/rewards/allocations/quadratic_funding.py

```diff
@@ -9,7 +9,8 @@
 class QuadraticFundingAllocations(ProjectAllocations):
     def _calc_allocations(self, allocations: List[AllocationItem]) -> Decimal:
         sum_sqrts = sum(
-            [Decimal(sqrt(allocation.amount)) for allocation in allocations]
+            Decimal(sqrt(allocation.uq_score * allocation.amount))
```

Comment:
can't `uq_score` be `None` here? What if we run an allocation simulation before first allocation, so there is no uq score in db yet?